TM1py/Objects/MDXView.py

```diff
@@ -15,13 +15,39 @@ class MDXView(View):
         IMPORTANT. MDXViews can't be seen through the old TM1 clients (Archict, Perspectives). They do exist though!
     """
 
-    def __init__(self, cube_name: str, view_name: str, MDX: str):
+    def __init__(self, cube_name: str, view_name: str, MDX: str, Meta: dict = {}):
         View.__init__(self, cube_name, view_name)
         self._mdx = MDX
+        self._aliases = Meta.get('Aliases', {})
```

Comment:
why not store the `meta` as a property in the class? Perhaps there is more interesting stuff in it?